File: Models/Entities/Serving.cs
using DAWProject.Models.Base;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace DAWProject.Models.Entities
{
	public class Serving : BaseEntity
	{
		public Double Quantity { get; set; }
		public Double Calories { get; set; }
		public int AlimentId { get; set; }
		public Aliment Aliment { get; set; }
		public int MealId { get; set; }
		public Meal Meal { get; set; }
	}
}

File: ClientApp/src/app/models/serving.ts
import { Aliment } from "./aliment";
import { Meal } from "./meal";

export class Serving {

  id: number;
  quantity: number;
  calories: number;
  alimentid: number;
  aliment: Aliment;
  mealid: number;
  meal: Meal;

  constructor(input?: any) {
    Object.assign(this, input);

  }

}
